xdrip/Storyboards/‏‏ar.lproj/SettingsViews.strings

```diff
@@ -1,79 +1,79 @@
-"displayDeltaInCalendarEvent" = "Display Delta?";
-"infoCalendarAccessDeniedByUser" = "You previously denied access to your Calendar.\n\nTo enable it go to your device settings, privacy, calendars and enable it.";
-"infoCalendarAccessRestricted" = "You cannot give authorization to xDrip4iOS to access your calendar. This is possibly due to active restrictions such as parental controls being in place.";
-"writeTraceToFile" = "Write Trace to File?";
-"sectionTitleTrace" = "Trace";
-"sendTraceFile" = "Send Trace File";
-"describeProblem" = "Explain why you need to send the trace file with as much detail as possible. If you have already reported your problem in the Facebook support group 'xDrip4iOS', then mention your facebook name in the e-mail";
-"emailbodyText" = "Problem Description: ";
-"failedToSendEmail" = "Failed to Send Email";
+"settingsviews_giveNightScoutAPIKey" = "أدخل الـ API_SECRET الخاص بك";
+"editScheduleTimePickerSubtitle" = "تغيير: ";
+"timeScheduleViewTitle" = "جدولة وقت تفعيل/إغلاق لـ";
+"uploadSensorStartTime" = "تحميل بداية وقت الحساس؟";
+"testUrlAndAPIKey" = "اختبار رابط منصة نايتسكاوت و API_SECRET؟ ";
+"settingsviews_sectiontitlespeak" = "الصوت";
+"settingsviews_speakBgReadings" = "نطق قراءات السكر؟";
+"settingsviews_speakBgReadingslanguage" = "اللغة:";
+"settingsviews_speakreadingslanguageselection" = "اختر اللغة";
+"settingsviews_speakTrend" = "نطق الاتجاه؟";
+"settingsviews_speakDelta" = "نطق الفرق؟";
+"settingsviews_speakInterval" = "الفاصل الزمني:";
+"settingsviews_speakIntervalMessage" = "الفاصل الزمني الأدنى بين قراءتين بالدقائق";
+"settingsviews_Version" = "الإصدار:";
+"settingsviews_license" = "رخصة";
+"settingsviews_build" = "رقم الإصدار:";
+"m5stack_settingsviews_settingstitle" = "إعدادات M5 Stack";
+"m5stack_settingsviews_textColor" = "لون الخط";
+"m5stack_settingsviews_backGroundColor" = "لون الخلفية";
+"m5stack_settingsviews_sectiontitlebluetooth" = "بلوتوث";
+"m5stack_settingsviews_giveBluetoothPassword" = "أدخل كلمة المرور للبلوتوث";
+"m5stack_settingsviews_brightness" = "سطوع الشاشة";
+"appleWatchSectionTitle" = "ساعة أبل";
+"createCalendarEvent" = "إنشاء أحداث التقويم؟";
+"calenderId" = "التقويم للاستخدام؟";
+"displayTrendInCalendarEvent" = "عرض الإتجاه؟";
+"displayUnitInCalendarEvent" = "عرض الوحدة؟";
+"displayDeltaInCalendarEvent" = "عرض الفرق؟";
+"infoCalendarAccessDeniedByUser" = "لقد رفضت الوصول إلى التقويم الخاص بك مسبقًا.\n\nلتمكينه ، انتقل إلى إعدادات جهازك ثم الخصوصية ثم التقويم ثم قم بتمكينه.";
+"infoCalendarAccessRestricted" = "لا يمكنك منح إذن لـ xDrip4iOS للوصول إلى التقويم الخاص بك. قد يكون هذا بسبب القيود النشطة مثل وجود أدوات الرقابة الأبوية.";
+"writeTraceToFile" = "كتابة التتبع إلى ملف؟";
+"sectionTitleTrace" = "تتبع";
+"sendTraceFile" = "إرسال ملف التتبع";
+"describeProblem" = "اشرح سبب حاجتك لإرسال ملف التتبع بأكبر قدر ممكن من التفاصيل. إذا كنت قد ذكرت بالفعل مشكلتك في مجموعة دعم الفيسبوك "xDrip4iOS"، اذكر اسم الفيسبوك الخاص بك في البريد الإلكتروني";
```

Comment:
when quoting "xdrip4iOS"  can you replace the double quote by single quote ?

Reply:
did it